Tests/Security/FormAuthenticatorTest.php

```diff
@@ -54,10 +54,8 @@ public function testCheckCredentials()
      */
     public function testBadCredentialsException()
     {
-        $this->expectException(
-            'Symfony\Component\Security\Core\Exception\BadCredentialsException',
-            $this->formAuthenticator->checkCredentials($this->getCredentials('wouter', 'wrongPassword'), $this->getUser())
-        );
+        $this->setExpectedException('Symfony\Component\Security\Core\Exception\BadCredentialsException');
+        $this->formAuthenticator->checkCredentials($this->getCredentials('wouter', 'wrongPassword'), $this->getUser());
```

Comment:
let ook best op je line lenghts. De maatstaaf is 80, de soft limit is 120. Deze lijn is zeker boven de 80 en kan eenvoudig er onder gehouden worden.